src/modules/roc_packet/target_stdio/roc_packet/address_to_str.cpp

```diff
@@ -19,27 +19,93 @@ address_to_str::address_to_str(const Address& addr) {
 
     switch (addr.version()) {
     case 4: {
-        if (!addr.get_ip(buffer_, sizeof(buffer_))) {
+        if (!addr.get_host(buffer_, sizeof(buffer_))) {
             roc_log(LogError, "address to str: can't format ip");
+            strcpy(buffer_, "error");
```

Comment:
Could you please replace `error` and `none` with `<error>` and `<none>`, similar to https://github.com/roc-project/roc/blob/develop/src/modules/roc_address/io_uri_to_str.cpp#L14-L18 ?

Reply:
Done